argoproj-crd/templates/applications.yaml

```diff
@@ -10,6 +10,11 @@ metadata:
     {{- with $app.labels }}
     {{- toYaml . | nindent 4 }}
     {{- end }}
+  annotations:
+  {{- if $app.chatworkRoomId }}
+    recipients.argocd-notifications.argoproj.io: webhook:chatwork
+    chatwork: "{{ $app.chatworkRoomId }}"
+  {{- end }}
```

Comment:
@taishin 
I think it's better to be able to inject `$app.annotations`.
It's more generic that way.

Reply:
@taishin 
I don't think it's necessary to have both `if` and `with`.